localization/english/wc_buildings_l_english.yml

```
@@ -1,11 +1,55 @@
+ 
+ building_type_lordaeron_palace_01:0 "Lordaeron Palace"
+ building_type_lordaeron_palace_01_desc:1 "A palace that holds together the domain of its ruler. It is a beacon of prosperity and unity to all who lay eyes upon it." 
+ building_lordaeron_palace_01:1 "Lordaeron Palace"
+ building_lordaeron_palace_01_desc:1 "The Palace of Lordaeron is the jewel of the Empire of Lordaeron and shows just how far humans have come. Its presence exudes an aura of prestige and with a hint of decadence."
+ 
+ building_type_temple_of_atal_hakkar_01:0 "Temple of Atal'Hakar"
+ building_type_temple_of_atal_hakkar_01_desc:1 "A Temple built to bring back an ancient blood god into Azeroth. A massive shrine to his greatness."
+ building_temple_of_atal_hakkar_01:1 "Temple of Atal'Hakar"
+ building_temple_of_atal_hakkar_01_desc:1 " An influential group of troll priests, known as the Atal'ai, seek to bring back an ancient blood god named Hakkar the Soulflayer. They erected this great temple to Hakkar - where they could prepare for his arrival into the physical world."
+ 
+ building_type_thoradins_wall_01:0 "Thoradin's Wall"
+ building_type_thoradins_wall_01_desc:1 "Colossal walls that sheild the denzins of its holders domain. All unwanted guests must firstly face this almost insurmountable obstacle."
+ building_thoradins_wall_01:1 "Ruined Thoradin's Wall"
+ building_thoradins_wall_01_desc:1 "Once a wonder for our people, a testament to our power and our technological prowess, Thoradin's Wall now only statnds as a symbol of our downfall. Some parts of the wall still remain and help keep the peace, but the mostly ruined wall is but a shell of the once great shield against those who would wish harm upon us."
+ building_thoradins_wall_02:1 "Thoradin's Wall"
+ building_thoradins_wall_02_desc:1 "In its full glory, Thoradin's Wall stands proudly as a bastion for our people. We are protected by the towering walls and the brave soldiers guarding us from the dangers outside."
+ 
+ building_type_ruined_palace_01:0 "Ruined Palace"
+ building_type_ruined_palace_01_desc:1 "A city that spans beneath much of Lordaeron's ruin. A grave to kingdom."
+ building_ruined_palace_01:1 "Ruined Palace"
+ building_ruined_palace_01_desc:1 "The Ruins of Lordaeron is all that remains of the city of Lordaeron, the great capital city of the Kingdom of Lordaeron. Now it serves as a sad reminder of the power of the Scourge. The Ruined Palace still stands and its inhabitants are constantly haunted by the final tragic events that led to the death of King Terenas and the city's destruction."
+ 
+ building_type_sunwell_01:0 "The Sunwell"
+ building_type_sunwell_01_desc:1 "A fountain of mystical power located in the Sunwell Plateau on the Isle of Quel'Danas. Its presence in its users lives is comparable to eating or breathing."
```

```suggestion
 building_type_sunwell_01_desc:1 "A fountain of mystical power located within the Sunwell Plateau on the Isle of Quel'Danas. Its users rely on its energies for sustenance."
```

I think this part of the sentence sounded a bit confusing. Not sure if I made it better or worse though.